tests/lib/rules/require-prop-comment.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * @author CZB
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/require-prop-comment')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('require-prop-comment', rule, {
+  valid: [
+    {
+      code: `
+      <script>
+      import { defineComponent } from '@vue/composition-api'
+
+      export default defineComponent({
+        props: {
+          /**
+           * a comment
+           */
+          a: Number,
+        }
+      })
+      </script>
+      `
+    },
+    {
+      code: `
+      <script setup>
+      const props = defineProps({
+        /*
+         * a comment
+         */
+        a: Number,
+        /**
+         * b comment
+         */
+        b: Number
+      })
+      </script>
+      `,
+      options: [{ type: 'block' }]
+    },
```

Comment:
I've changed the implementation such that JSDoc comments are not considered block comments anymore. Please fix this test accordingly.

Reply:
I noticed this change：comment.value.charAt(0) === '*'
In JSDoc documents
JSDoc comments should generally be placed immediately before the code being documented. Each comment must start with a /** sequence in order to be recognized by the JSDoc parser. Comments beginning with /*, /***, or more than 3 stars will be ignored. This is a feature to allow you to suppress parsing of comment blocks.

Reply:
You're right, I fixed it in [vuejs/eslint-plugin-vue@`73ebb8e` (#2019)](https://github.com/vuejs/eslint-plugin-vue/pull/2019/commits/73ebb8ef34a50fe6b1f313fb4390742ee7f1756c).